force-app/main/default/classes/NoteController.cls

```diff
@@ -53,29 +62,32 @@ public with sharing class NoteController {
     }
 
     @AuraEnabled
-    public static Note__c updateNote(String noteId, String changeLabelValue, String changeDescriptionValue){
+    public static Note__c updateNote(Id noteId, String changeLabelValue, String changeDescriptionValue){
         
         System.debug('Method updateNote: ============');
-        System.debug('nnteId: ' + noteId);
+        System.debug('noteId: ' + noteId);
         System.debug('changeLabelValue: ' + changeLabelValue);
         System.debug('changeDescriptionValue:' + changeDescriptionValue);
         
         
```

Comment:
if you have Id of some record you can write like a new record:
String accountId = '001dsdsdfs';
Account a = new Account(Id = accountId, someField__c = 'new updated value');
update a;